synapse/tests/utils.py

```diff
@@ -1192,12 +1192,15 @@ def getAsyncLoggerStream(self, logname, mesg=''):
         handler = logging.StreamHandler(stream)
         slogger = logging.getLogger(logname)
         slogger.addHandler(handler)
+        level = slogger.level
+        slogger.setLevel('INFO')
```

Comment:
If we're here - lets set the logger to DEBUG and make sure we also make the change to ``getLoggerStream`` ?